guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc

```diff
@@ -1,5 +1,5 @@
-[id="Adding_Custom_DEB_Repository_Example_for_Debian_11_{context}"]
-= Adding {customdebtitle} repository example for Debian 11
+[id="Adding_Custom_Deb_Repository_Example_for_Debian_11_{context}"]
+= Adding Deb repository example for Debian 11
```

Comment:
The word "example" looks strange. I recommend changing to "Adding a Deb repository for Debian 11". 

Same for Ubuntu. Most of our procedures contain examples but they are still procedures.

Reply:
Reworded to "Adding upstream repositories for X", which IMHO describes best that it's about the default OS repos from upstream for the host OS.

Does this make sense to you?

Reply:
Much better!